src/main/kotlin/three/consulting/epoc/service/CustomerService.kt

```diff
@@ -6,11 +6,15 @@ import org.springframework.stereotype.Service
 import three.consulting.epoc.dto.CustomerDTO
 import three.consulting.epoc.entity.Customer
 import three.consulting.epoc.repository.CustomerRepository
+import mu.KotlinLogging
+
+private val logger = KotlinLogging.logger {}
```

Comment:
for some reason my IDEA is giving unresolved reference on both the `mu` of the import and `KotlinLogging` from initialisation. But everything seems to work.

Reply:
Idea sometimes takes a while to notice new changes, but it should resolve with either time or reindexing gradle dependencies.

And yeah you can add logging to this service class in this pr. At least to when functions are called and when exceptions are thrown. In the latter case try to include thrown exception in the log. Kotlinlogging is just a wrapper for sjf4j where it's possible to pass exception as a comma separated argument to the logger call, so try something similar.